test/ssmParameter.go

```diff
@@ -0,0 +1,54 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build integration
+// +build integration
+
+package test
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	"github.com/aws/aws-sdk-go-v2/service/ssm/types"
+)
+
+var (
+	ssmCtx    context.Context
+	ssmClient *ssm.Client
+)
+
+func PutStringParameter(name, value string) error {
+	return putParameter(name, value, types.ParameterTypeString)
+}
+
+func putParameter(name string, value string, paramType types.ParameterType) error {
+	svc, ctx, err := getSsmClient()
+	if err != nil {
+		return err
+	}
+	isOverwriteAllowed := true
+
+	_, err = svc.PutParameter(ctx, &ssm.PutParameterInput{
+		Name:      aws.String(name),
+		Value:     aws.String(value),
+		Type:      paramType,
+		Overwrite: &isOverwriteAllowed,
+	})
+
+	return err
+}
+
+func getSsmClient() (*ssm.Client, context.Context, error) {
+	if ssmClient == nil {
+		ssmCtx = context.Background()
+		cfg, err := config.LoadDefaultConfig(ssmCtx)
```

Comment:
IMO, the others client (e.g [cloudwatch](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/test/cwm_util.go#L73)) should share the same configuration without any different (e.g credentials, etc)

Reply:
You want me to pull existing stuff out?